charts/internal/calico/templates/custom-resource-definition/crd-bgpconfigurations.yaml

```diff
@@ -1,15 +1,135 @@
 ---
-apiVersion: apiextensions.k8s.io/v1beta1
+apiVersion: apiextensions.k8s.io/v1
```

Comment:
This means that we cannot install calico on clusters < 1.16 anymore. The `apiextensions.k8s.io/v1` API group is only available in 1.16+.
Probably this calico version is not compatible with Kubernetes < 1.16, or? Can you check this? If yes we probably have to stay with 3.13 for Kubernetes < 1.16, and use the 3.16 only for Kubernetes >= 1.16.

Reply:
That is a good point, will check that.